app/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_add_zeros_if_arguments_zeros():
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_should_return_zeros_if_age_less_fifteen():
+    assert get_human_age(14, 14) == [0, 0]
+
+
+def test_should_return_one_if_age_equals_fifteen():
+    assert get_human_age(15, 15) == [1, 1]
+
+
+def test_twenty_four_years_should_convert_into_two_human_years():
+    assert get_human_age(24, 24) == [2, 2]
```

Comment:
test the top border the same as you done for [0, 0]

Reply:
What's the top border?